### PR TITLE
adding KS, AD, CvM tests

### PR DESCRIPTION
**Pull request by ih64**
some tests for the PIT histograms. These are the Kolmogrov-Smirnov (KS) test, Cramer-von Misses statistic (CvM) and Anderson-Darling statistic (AD)

### CODE REVIEW

File: pzblend/pzblend.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import gzip
 import joblib
 import random
+import skgof
```
Comment (enourbakhsh) [inline pzblend/pzblend.py:23]: I think we should add `skgof` to `install_requires` in `setup.py` then.

**Review by enourbakhsh — CHANGES_REQUESTED** [review]

File: pzblend/pzblend.py (hunk 1 above)
Comment (ih64) [reply pzblend/pzblend.py:23]: sounds good!

File: pzblend/pzblend.py [hunk 2]
```diff
@@ -553,6 +554,27 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
         else:
             if verbose: logging.info(f'{inspect.stack()[1].function}:{inspect.stack()[0].function}: The PIT values remained unchanged since no update was needed.')
               
+    def KS(self):
+        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
```
Comment (enourbakhsh) [inline pzblend/pzblend.py]: pit_hist() should be calc_pits()

File: pzblend/pzblend.py [hunk 3]
```diff
@@ -553,6 +554,27 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
...
+        pits = np.array(self.PITS)
+        ks_result = skgof.ks_test(pits, stats.uniform())
+        return ks_result.statistic, ks_result.pvalue
+
+    def CvM(self):
+        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
```
Comment (enourbakhsh) [inline pzblend/pzblend.py]: pit_hist() should be calc_pits()

File: pzblend/pzblend.py [hunk 4]
```diff
@@ -553,6 +554,27 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
...
+        pits = np.array(self.PITS)
+        cvm_result = skgof.cvm_test(pits, stats.uniform())
+        return cvm_result.statistic, cvm_result.pvalue
+
+    def AD(self, vmin=.005, vmax=.995):
+        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
```
Comment (enourbakhsh) [inline pzblend/pzblend.py]: pit_hist() should be calc_pits()

**Comment by cosmicshear** [comment]
I had some minor comments, otherwise it looks good!

**Review by enourbakhsh — CHANGES_REQUESTED** [review]
I had some minor comments on the code, otherwise it looks good! Thanks Imran!

**Review by enourbakhsh — APPROVED** [review]
It looks good.

File: pzblend/pzblend.py [hunk 5]
```diff
@@ -555,19 +555,19 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
             if verbose: logging.info(f'{inspect.stack()[1].function}:{inspect.stack()[0].function}: The PIT values remained unchanged since no update was needed.')
               
     def KS(self):
-        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
+        assert self.PITS is not None, "you must create pit_array first, try running calc_pit()"
```
Comment (cosmicshear) [inline pzblend/pzblend.py:558]: calc_pits()

File: pzblend/pzblend.py [hunk 6]
```diff
@@ -555,19 +555,19 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
...
         pits = np.array(self.PITS)
         ks_result = skgof.ks_test(pits, stats.uniform())
         return ks_result.statistic, ks_result.pvalue
 
     def CvM(self):
-        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
+        assert self.PITS is not None, "you must create pit_array first, try running calc_pit()"
```
Comment (enourbakhsh) [inline pzblend/pzblend.py]: calc_pits()

File: pzblend/pzblend.py (hunk 5 above)
Comment (enourbakhsh) [inline pzblend/pzblend.py]: calc_pits()

File: pzblend/pzblend.py [hunk 7]
```diff
@@ -555,19 +555,19 @@ def calc_pits(self,leave=False,verbose=True,force_refresh=False):
...
         pits = np.array(self.PITS)
         cvm_result = skgof.cvm_test(pits, stats.uniform())
         return cvm_result.statistic, cvm_result.pvalue
 
     def AD(self, vmin=.005, vmax=.995):
-        assert self.PITS is not None, "you must create pit_array first, try running pit_hist()"
+        assert self.PITS is not None, "you must create pit_array first, try running calc_pit()"
```
Comment (enourbakhsh) [inline pzblend/pzblend.py]: calc_pits()

**Review by enourbakhsh — CHANGES_REQUESTED** [review]
Caught typos.

**Review by enourbakhsh — APPROVED** [review]

**Review by enourbakhsh — APPROVED** [review]

**Pull request merged**